docs/topics/modules/creating_new_space.adoc

```diff
@@ -17,7 +17,7 @@ endif::[]
 [discrete]
 == Procedure
 
-Create a new space as follows:
+Create a new space in {osio} as follows:
```

Comment:
Minor quibble, but is it necessary to mention osio here, since this is the middle of the module and the next line does mention osio again.